week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,14 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  var result = price * 1.4 + 0.01 + " $ ";
+  return result;
+}
+function convertToBRL(price) {
+  var result = price * 1.4 + " B$";
+  return result;
+}
```

Comment:
In the instructions it mentions the "DRY principle". That means [don't repeat yourself](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Another way to solve this problem is to write a separate function for adding the foreign transaction fee. That would look like this:

```
function convertToUSD(price) {
  return price * 1.4;
}

function addForeignFee(price) {
  return price * 0.01;
}
```

Then, to get the result, you would do this:

```
var price = 12.5
var priceInUSD = covertToUSD(price);
var finalUSDPrice = addForeignFee(priceInUSD);
```